src/System.Buffers.Primitives/System/Buffers/IOutput.cs

```diff
@@ -3,12 +3,26 @@
 
 namespace System.Buffers
 {
+    /// <summary>
+    /// Represents a <see cref="Byte"/> sink
+    /// </summary>
     public interface IOutput
     {
+        /// <summary>
+        /// Notifies <see cref="IOutput"/> that <see cref="bytes"/> amount of data was written to <see cref="Span{Byte}"/>/<see cref="Memory{Byte}"/>
```

Comment:
paramref instead of cref for "bytes"

Here and elsewhere.
I believe the pattern is to use paramref for parameter names, and cref for types, methods, and properties.